04_environment/index.tex

```diff
@@ -160,26 +145,30 @@ \subsubsection{Incorporated Changes: Population Density}
 
-Finally, more foraging options could be added in future implementations. For example, a third foraging technique, call it whale it hunting, could be created. That would have an even higher return than the deer hunting, deeming it an appealing forage method. The barriers of entry could be similar to that of the deer hunting method in this scenario but at the same time, there would be a more spaced out/expensive tier system. This combination would potentially incentivise cooperation between islands, if their strategy was to maximise returns. Also, as additional foraging methods have been introduced, the type of return has been fixed for each foraging method and observe how agents decide to behave. For example, the deer and fish hunting methods result in returns proportional to inputs, whereas the whale hunting method would result in an equal split of returns between all participating islands. Would this mean that islands choose a fairer division of returns and not go on whale foraging at all? Or Would they be intrigued by the higher costs and choose whale hunting over the other foraging methods? In the latter question agents would need to take in mind that an island may be entering an insignificant amount reaping however, a large reward. It is therefore up to the islands to decide if the higher payoff is worth the risk of other islands free-riding.
+Finally, more foraging options could be added in future implementations. For example, a third foraging technique; "whale hunting"could be created. That would have an even higher return than the deer hunting, deeming it an appealing forage method. The barriers of entry could be similar to that of the deer hunting method in this scenario but at the same time, there would be a more spaced out/expensive tier system. This combination would potentially incentivise cooperation between islands, if their strategy was to maximise returns. Also, as additional foraging methods have been introduced, the type of return has been fixed for each foraging method and observe how agents decide to behave. For example, the deer and fish hunting methods result in returns proportional to inputs, whereas the whale hunting method would result in an equal split of returns between all participating islands. Would this mean that islands choose a fairer division of returns and not go on whale foraging at all? Or Would they be intrigued by the higher costs and choose whale hunting over the other foraging methods? In the latter question agents would need to take in mind that an island may be entering an insignificant amount reaping. However, a large reward. It is therefore up to the islands to decide if the higher payoff is worth the risk of other islands free-riding.
```

Comment:
**whale hunting** could be created --> no quotation marks AND put some space!

 That would have an even higher return   ----> This option would have..  
 'That' sounds bad in the way you've written

Reply:
suggested re-writing:

**Or would** they be intrigued by the higher costs and choose whale hunting over the other foraging methods? In the **latter question,** **the islands would need to take into consideration that ....**

what you mean by reaping ?? revise it?

You CANNOT write a sentence like: "However, a large reward." :( just looks bad.